capellambse/_diagram_cache.py

```diff
@@ -324,7 +324,7 @@ def _text_extents(element: etree._Element) -> Extents:
     x = float(element.get("x", 0))
     y = float(element.get("y", 0))
     text = element.text or ""
-    font_size_str = element.get("font-size", "12").strip().lower()
+    font_size_str = element.get("font-size", "10").strip().lower()
```

Comment:
I think it'd make sense to also use a module-level constant here, derived from `helpers.DEFAULT_FONT_SIZE`.